### PR TITLE
Fix PEP warnings in testing-unittest-py

### PR DESCRIPTION
**Pull request by ocworld**
The pr is to fix warnings In testing-unittest-py.
Firstly, fixing warnings about PEP.
Secondly, removing unused import.

### CODE REVIEW

**Review by cnunciato — APPROVED** [review]
Thanks @ocworld!

**Comment by scottslowe** [comment]
@cnunciato The only failing tests are the DO and AWS tests that were fixed by #1497 (I kicked off the tests before that PR was merged). I'm OK to merge this if you are.

**Pull request merged**
